build.gradle

```diff
@@ -238,10 +236,10 @@ dependencies {
 
   implementation group: 'io.springfox', name: 'springfox-swagger2', version: versions.springfoxSwagger
 
-  implementation group: 'uk.gov.hmcts.reform', name: 'logging', version: versions.reformLogging
-  implementation group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: versions.reformLogging
+  implementation 'com.github.hmcts.java-logging:logging:5.1.7'
+  implementation 'com.github.hmcts.java-logging:logging-appinsights:5.1.7'
```

Comment:
can you upgrade versions.reformLogging to latest and use that instead of using hard code `5.1.7`